install.sh

```diff
@@ -1,32 +1,21 @@
 #!/bin/sh -e
 
-WORKSPACE="${GITHUB_WORKSPACE:=/usr/local}"
 WORKSPACE="/usr/local"
-VERSION=${version:=v0.1.0-alpha.2}
 REPO=${repo:=open-component-model/ocm}
-URL=https://github.com/$REPO/releases/download
-PLATFORM=linux-amd64
-if [ "$REPO" != "gardener/component-cli" ]; then
-  ARCHIVESUFFIX=.tgz
-  ARCHIVEFILE="ocm-$PLATFORM$ARCHIVESUFFIX"
-else
-  ARCHIVESUFFIX=.gz
-  ARCHIVEFILE="componentcli-$PLATFORM$ARCHIVESUFFIX"
-fi
-FILE="$(basename "$ARCHIVEFILE" $ARCHIVESUFFIX)"
+VERSION=${version:=v0.2.0-rc.1}
```

Comment:
@morri-son and I spoke earlier about cutting a release for `v0.2.0` today/soon so this RC version would become outdated soon... There is no harm in leaving this for now but we probably want to add a way to update the version automatically when there is a new OCM CLI release, right?